File: retirement_calculator/retirement_calculator.rb
require 'Date'

puts "What is your current age?"

current_age_input = gets.chomp

puts "At what age would you like to retire?"

age_to_retire_input = gets.chomp

current_age = current_age_input.to_i
age_to_retire = age_to_retire_input.to_i

years_to_retire = age_to_retire - current_age

puts "You have #{years_to_retire} to retire."

current_year = Date.today.year

puts "It's #{current_year}, so you can retire in #{current_year + years_to_retire}"

File: retirement_calculator/python/retirement_calculator.py
import datetime

now = datetime.datetime.now()

current_age = int(input('What is your current age? '))
age_to_retire = int(input('At what age would you like to retire? '))

years_to_retire = age_to_retire - current_age

print('You have {:d} years to retire'.format(years_to_retire))

current_year = now.year

print('It\'s {:d},'.format(current_year), 'so you can retire in {:d}'.format(current_year + years_to_retire))